en/components/pivotGrid/pivot-grid-custom.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Angular Pivot Grid Custom Remote | Pivot Tables | Infragistics
+_description: Create fast, responsive Angular pivot grids and tables with Ignite UI for Angular. Perform complex data analysis via pivot data.
+_keywords: angular pivot grid, angular material pivot table, ignite ui for angular, pivot grid customization, pivot grid remote, pivot remote
+---
+
+
+# Remote Operations
+
+In scenarios where the pivot data is already grouped and aggregated from a remote service and there's no need for further processing on the client, the pivot grid can be configured to use a custom empty strategy that will skip data processing on the client and allow it to directly display the data as is:
+
+```typescript
+public pivotConfigHierarchy: IPivotConfiguration = {
+    columnStrategy: NoopPivotDimensionsStrategy.instance(),
+    rowStrategy: NoopPivotDimensionsStrategy.instance(),
+}
+```
+
+Users have the ability to achieve certain scenarios by feeding the pivot grid with already aggregated data.
+There are some requirements on how the data should look like and some specifics regarding hierarchies in the pivot view. For example, to declare hierarchy in `rows` dimension:
+
+```typescript
+rows: [
+    {
+        memberName: 'AllProducts',
+        memberFunction: () => 'All Products',
+        enabled: true,
+        childLevel: {
+            memberName: 'ProductCategory',
+            enabled: true
+        }
+    }
+]
+```
+
+And an example of the aggregated would be:
+
+```typescript
+public aggregatedData = [
+    {
+        ProductCategory: 'All', AllProducts: 'All Products', All: 1000, 'All-Bulgaria': 774, 'All-USA': 829, 'All-Uruguay': 524, AllProducts_records: [
```

Comment:
This gives just a single example of how to setup `_records`. The other properties are given little to no explanation. Ideally you should expand this portion a lot with examples either using screenshots or a sample that has a state described by sample json like this but also clearly showing how certain properties affect the view.

Reply:
Added more details on the pivot keys.